org.gnome.gbrainy.json

```diff
@@ -3,56 +3,68 @@
     "sdk": "org.gnome.Sdk",
     "branch": "stable",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.36",
+    "runtime-version": "41",
     "command": "gbrainy",
     "rename-appdata-file": "gbrainy.appdata.xml",
     "rename-desktop-file": "gbrainy.desktop",
     "rename-icon": "gbrainy",
     "copy-icon": true,
-    "sdk-extensions": ["org.freedesktop.Sdk.Extension.mono5"],
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.mono6"
+    ],
     "finish-args": [
         "--share=ipc",
         "--share=network",
         "--socket=wayland",
         "--socket=pulseaudio",
         "--socket=x11"
     ],
-    "build-options" : {
-        "append-path": "/usr/lib/sdk/mono5/bin",
-        "append-ld-library-path": "/usr/lib/sdk/mono5/lib",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/mono6/bin",
+        "append-ld-library-path": "/usr/lib/sdk/mono6/lib",
         "env": {
-            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono5/lib/pkgconfig"
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono6/lib/pkgconfig"
```

Comment:
`"append-pkg-config-path": "/usr/lib/sdk/mono6/lib/pkgconfig"` is probably better here.

Reply:
just to clarify it would be one level up out of the `env` object.